### PR TITLE
Added two new files, modified another_file

### PR DESCRIPTION
**Pull request by annaoliinyk**
The task was done:
- switched to new branch
- another_file.txt modified;
- 2 new files were added

### CODE REVIEW

File: About pull request merges.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+The pull request has merge conflicts.
+Rebasing the commits from the base branch into the head branch runs into conflicts.
+Rebasing the commits is considered "unsafe," such as when a rebase is possible without merge conflicts but would produce a different result than a merge would.
+If you still want to rebase the commits but can't rebase and merge automatically on GitHub.com you must:
+
+Rebase the topic branch (or head branch) onto the base branch locally on the command line
+Resolve any merge conflicts on the command line.
+Force-push the rebased commits to the pull request's topic branch (or remote head branch).
+Anyone with write permissions in the repository, can then merge the changes using the rebase and merge button on GitHub.com.
```
Comment (AnaKovaGH) [inline About pull request merges.txt:22]: Please add empty line in the end of the file

File: Managing a branch protection rule.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+You can create a branch protection rule to enforce certain workflows for one or more branches, such as requiring an approving review or passing status checks for all pull requests merged into the protected branch.
```
Comment (AnaKovaGH) [inline Managing a branch protection rule.txt:1]: Same here

File: another_file.txt [hunk 3]
```diff
@@ -1 +1,2 @@
 No matter what's here 
+Will add more soon
```
Comment (AnaKovaGH) [inline another_file.txt]: Same here

**Review by AnaKovaGH — CHANGES_REQUESTED** [review]
Also please add .gitignore file.

**Comment by annaoliinyk** [comment]
The changes were added, creating a new pull request

**Pull request closed without merging**